### DISCUSSION TITLE
Block Size

### DISCUSSION THREAD

**Discussion by andreslomelig** [pyopencl/pyopencl#790] [Troubleshooting] [answered]
Hi, I 'am new with this, is there any way to control the work group sizes when using them?

**Comment by inducer** [comment, marked as answer]
Third argument to the kernel call:

https://github.com/inducer/pyopencl/blob/5085bb3a046177cd21c489636e3dbc3ac59b94ea/examples/demo.py#L30

(`None` here.)
